templates/frontend/objects/issue_toc.tpl

```diff
@@ -41,12 +41,10 @@
 		{* Issue cover image *}
 		{assign var=issueCover value=$issue->getLocalizedCoverImageUrl()}
 		{if $issueCover}
-			<a class="cover" href="{url op="view" page="issue" path=$issue->getBestIssueId()}">
```

Comment:
Have you tested these changes out to see how they change the layout? The default theme has styles attached to the `.cover` class and which expect too find `<a class="cover"><img></a>` in this code base. The styles may need to be updated. See https://github.com/pkp/ojs/blob/main/plugins/themes/default/styles/objects/issue_summary.less and https://github.com/pkp/ojs/blob/main/plugins/themes/default/styles/objects/issue_toc.less for example.

Reply:
I may have overlooked this class. Let me run a few more tests.

Reply:
I've adjusted the HTML replacing the `A HREF` by a `DIV` tag and preserving the `css` class. It is the same approach currently applied to list articles with cover images. The build test has failed though.
Could you please review it @NateWr and let me know if it is correct now?